cfgov/ask_cfpb/scripts/export_ask_data.py

```diff
@@ -61,7 +63,10 @@ def assemble_output():
         output['Language'] = page['language']
         output['RelatedResource'] = page['related_resource__title']
         output['Question'] = page['question'].replace('\x81', '')
-        answer_streamfield = page['answer_content'].stream_data
+        if wagtail.VERSION < (2, 12):  # pragma: no cover
```

Comment:
Since you've settled on just pinning Wagtail to 2.12.3 in `requirements/wagtail.txt`, this doesn't seem necessary.

Reply:
Agreed.